docs/.gitignore

```diff
@@ -0,0 +1,2 @@
+
```

Comment:
The default `sncli/.gitignore` already includes `build`. So leaving the Makefile at default and using `-BUILDDIR = build` would do the same thing as adding this file and then editing the Makefile.